src/Documenter.jl

```diff
@@ -573,28 +655,30 @@ function deploydocs(;
     end
 
     # Check criteria for deployment
-    ## The deploydocs' repo should match TRAVIS_REPO_SLUG
-    repo_ok = occursin(travis_repo_slug, repo)
+    ## The deploydocs' repo should match the repo slug provided by CI
+    repo_ok = occursin(repo_slug, repo)
     ## Do not deploy for PRs
-    pr_ok = travis_pull_request == "false"
-    ## If a tag exist it should be a valid VersionNumber
-    tag_ok = isempty(travis_tag) || occursin(Base.VERSION_REGEX, travis_tag)
-    ## If no tag exists deploydocs' devbranch should match TRAVIS_BRANCH
-    branch_ok = !isempty(travis_tag) || travis_branch == devbranch
+    pr_ok = pull_request in ("false", "False") # Appveyor env vars are capitalized on Windows
+    ## If a tag exists, it should be a valid VersionNumber
+    tag_ok = isempty(tag) || occursin(Base.VERSION_REGEX, tag)
+    ## If no tag exists deploydocs' devbranch should match the CI branch
+    branch_ok = !isempty(tag) || branch == devbranch
     ## DOCUMENTER_KEY should exist
     key_ok = !isempty(documenter_key)
     ## Cron jobs should not deploy
-    type_ok = travis_event_type != "cron"
-    should_deploy = repo_ok && pr_ok && tag_ok && branch_ok && key_ok && type_ok
+    type_ok = event_type != "cron"
+
+    should_deploy = repo_ok && pr_ok && tag_ok && branch_ok && key_ok && type_ok && finaldeploy
 
     marker(x) = x ? "✔" : "✘"
     @info """Deployment criteria:
-    - $(marker(repo_ok)) ENV["TRAVIS_REPO_SLUG"]="$(travis_repo_slug)" occurs in repo="$(repo)"
-    - $(marker(pr_ok)) ENV["TRAVIS_PULL_REQUEST"]="$(travis_pull_request)" is "false"
-    - $(marker(tag_ok)) ENV["TRAVIS_TAG"]="$(travis_tag)" is (i) empty or (ii) a valid VersionNumber
-    - $(marker(branch_ok)) ENV["TRAVIS_BRANCH"]="$(travis_branch)" matches devbranch="$(devbranch)" (if tag is empty)
+    - $(marker(repo_ok)) CI repo slug = "$(repo_slug)" occurs in repo = "$(repo)"
+    - $(marker(pr_ok)) CI pull request indicator = "$(pull_request)" is "false"
```

Comment:
To be consistent with the `pr_ok = ` line above:
```suggestion
    - $(marker(pr_ok)) CI pull request indicator = "$(pull_request)" is "false" or "False"
```

Could also maybe `lowecase` `pull_request` and say it's case-insensitive.